tests/basic/basic_test.go

```diff
@@ -30,6 +31,12 @@ var _ = BeforeSuite(func() {
 
 // This test performs basic test of starting an application and destroying it (along with storage)
 var _ = Describe("{SetupTeardown}", func() {
+	var testrailId = 35258
+	// testrailId corresponds to: https://portworx.testrail.net/index.php?/cases/view/35258
+	var runId int
+	JustBeforeEach(func() {
+		runId = testrailuttils.AddRunsToMilestone(testrailId)
```

Comment:
use `runID :=` and remove `var runId int`

Reply:
this runId is used outside JustBeforeEach method as well. So we need to declare the variable outside the method.